accountsplus/views.py

```diff
@@ -172,6 +172,14 @@ def password_reset(request,
                    html_email_template_name=None,
                    extra_email_context=None):
     User = django.contrib.auth.get_user_model()
+    # We set this always in the middleware to the preferred language of the user
+    if request.method == 'POST':
+        email = request.POST['email']
+        try:
+            user = User.objects.get(email=email)
+            extra_email_context['email_lang'] = user.preferred_language or request.LANGUAGE_CODE
```

Comment:
@nasief you may add `email_lang` to the `extra_context` parameter. Is there a reason why you need this `extra_email_context`

Reply:
@waseem-omar `extra_context` is not supplied to the sending email part of the Form `PasswordChangeForm` where email is sent. I went through the code and I found that `extra_email_context` is the context used with the `send` method.